README.md

```diff
@@ -19,24 +24,25 @@ and as the basis of the OpenDDS DCPS Information Repository (DCPSInfoRepo).
 
 The primary development of OpenDDS was done by
 [Object Computing, Incorporated](http://www.objectcomputing.com) in
-St. Louis and Phoenix.  It is released under the same generous license
-terms as ACE, TAO and MPC.  See the [LICENSE](LICENSE) file for details.
+St. Louis and Phoenix.  It is released under generous license
+terms similar to ACE, TAO and MPC.  See the [`LICENSE`](LICENSE) file for
+details.
 
 **Table of Contents:**
 
 * [Documentation](#documentation)
 * [Support](#support)
 * [Features](#features)
 * [Dependencies](#dependencies)
-  * [TAO](#tao)
+  * [ACE/TAO](#acetao)
   * [Perl](#perl)
+  * [Google Test](#google-test)
```

Comment:
Should this list mention that gtest is only needed if building tests?

Reply:
It says it in the actual content, this is just the table of contents.

Reply:
Right, I guess I was just concerned with someone scanning this list and using it as the guide for which dependencies are required.

Reply:
True, how about I merge it with rest of the optional list?